install/centos-installation-guide.md

```diff
@@ -29,28 +29,10 @@ $ sudo -E VERSION_ID=$VERSION_ID yum-config-manager --add-repo \
 $ sudo -E yum -y install kata-runtime kata-proxy kata-shim
 ```
 
-3. Configure Docker to use Kata Containers by default with the following commands:
+3. Install the upper layer (Docker or Kubernetes) and play with it
 
-```bash
```

Comment:
@sboeuf, @grahamwhaley, @egernst - I think the best way to resolve this issue is:

- Modify the existing install guides so that they **only** install the Kata packages.
- At the end of each install document add something like:

  > Decide which container manager you wish to use and click the corresponding link below:
  >
  > - [Docker](https://github.com/kata-containers/documentation/blob/master/install/centos-install-docker.md)
  > - [Kubernetes](https://github.com/kata-containers/documentation/blob/master/install/centos-install-docker.md)

  This would then allow the `kata-manager` utility to continue working and would also allow us to test both container manager install guides automatically in the various CIs.

  wdyt? I'm happy to create an initial PR based on my thoughts above. @sboeuf could then follow-up with a PR to add the `install/${distro}-install-kubernetes.md` files maybe?

Reply:
So, split the guide into the generics and then the two specifics for docker and k8s (and then - sub-split k8s maybe into cri-o and containerd installs?) - so we can auto test all the parts separately (or in a number of branched chains iyswim). Yes, that sounds right for the automation.

Reply:
yes. We're going to have to balance the following requirements:

- minimising duplication (easing maintenance for ourselves).
- ease-of-use and simple audit/comprehension for users.
- fully prescriptive install method (both for users and for the automation utilities).

Reply:
I think this is fine, so long as there is clear documentation on getting K8S setup with Kata, and that we don't duplicate too much.

Reply:
@jodh-intel ok let me remove the factorization commit here, we can follow up later once your PR landed.